package.json

```diff
@@ -24,17 +24,25 @@
   },
   "homepage": "https://github.com/Ryhthmic-Squad/grace-shopper#readme",
   "dependencies": {
+    "axios": "^0.21.1",
     "bcrypt": "^5.0.1",
     "express": "^4.17.1",
     "faker": "^5.5.3",
     "jsonwebtoken": "^8.5.1",
     "morgan": "^1.10.0",
     "pg": "^8.6.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
Correct me if I'm wrong, but with webpack transpiling all of our front-end I think axios and react should be dev-dependencies

Reply:
updated